adds opsgenie endpoint. (#424)
logzio/logz-docs: _source/user-guide/integrations/endpoints.md

Notification endpoints can receive notifications when an alert is triggered,
when a user shares a Kibana object,
or when Logz.io finds new insights in your logs.

Logz.io supports sending notifications to
Opsgenie, BigPanda, Datadog, PagerDuty, Slack, VictorOps, or your own custom endpoint.

> ![Notification endpoints](https://dytvr9ot2sszz.cloudfront.net/logz-docs/notification-endpoints/notification-endpoints.png)

Reviewer: ![image](https://user-images.githubusercontent.com/6845404/76897538-9c054780-689c-11ea-847b-a611cf0550f9.png)

This screen shot takes up most of the vertical space. Cutting it to the second row in the table should do the trick.

Author: Not sure what "cutting it to the second row" means. It is supposed to illustrate the page and also show the icons of the integrations we support "OOTB". I can rethink it.

Reviewer: It means to cut the image vertically, and show the first two rows of the table only. That should cut the vertical height by half and improve the flow of the page.

#### To manage notification endpoints

You can manage your endpoints
from [**Alerts & events > Alert endpoints**](https://app.logz.io/#/dashboard/alerts/endpoints).
You can also add new endpoints when you're configuring a new notification.

* To add a new endpoint, click **Add endpoint**,
  fill in the required information,
  and click **Save**.

  For example, to add an Opsgenie endpoint, you'll need to select **Opsgenie** from the dropdown list, name the endpoint, and fill in your Opsgenie API key. 
  

>   ![Opsgenie endpoints](https://dytvr9ot2sszz.cloudfront.net/logz-docs/notification-endpoints/opsgenie-endpoint.png)

Reviewer: ![image](https://user-images.githubusercontent.com/6845404/76897676-d969d500-689c-11ea-9d5d-8565601d7118.png)

This would be fine without the screen shot, and then you don't have to break up the bullet list.

Author: @imnotashrimp  - This screenshot was specifically requested by the PM. I could add a third section so as not to break up the list.

Reviewer: @shalper Yes, on second read - the bullet list should be reorganized anyway. Testing is part of the creation process - so those two bullets can be grouped together.

Then there'd still be the issue of having the screen shot and a note, one right on top of the other.

Setting a custom port as part of an endpoint URL
will result in alerts not being sent to said endpoint.
{:.info-box.important}

> * To test an endpoint after it is created, open it in edit mode. 

Reviewer: ```suggestion
* To test an endpoint after it's created, open it in edit mode. 
```

Contractions where they're possible :)

> Click **Run the test** and check the recepient for the test message. 

Reviewer: ```suggestion
Click **Run the test** and make sure the test message was received. 
```

"check the recipient" doesn't quite work here, but checking that it's received should do it.

Logz.io will notify you if the message couldn't be sent, 

> but you'll need to confirm that the message was received. 

Reviewer: ![image](https://user-images.githubusercontent.com/6845404/76897898-4b421e80-689d-11ea-93ff-f73345180258.png)

Move these above the note on line 46, so that the note doesn't break your bullet list.
